### PR TITLE
Added long_text option

### PR DESCRIPTION
**Pull request by 0atman**
Created a new 5000 character field

API PR https://github.com/alphagov/forms-api/pull/92

### CODE REVIEW

**Comment by DavidBiddle** [comment]
You can fix the broken Docker build by changing `nodejs=16.16.0-r0` --> `nodejs=16.17.1-r0` in the Dockerfile

**Comment by DavidBiddle** [comment]
There's no view for this field type in `app/views/question` - which one does it render? Let me know if you need an FE to add one

File: spec/models/question/long_text_spec.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      question = described_class.new
+      expect(question.show_answer).to eq ""
+    end
+  end
+
+  context "when given a string" do
+    it "validates without errors" do
+      question = described_class.new
+      question.validate
+      question.text = "testing"
```
Comment (aliuk2012) [inline spec/models/question/long_text_spec.rb:27]:
```suggestion
      question = described_class.new(text: "testing")
```

File: spec/models/question/long_text_spec.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      question.validate
+      expect(question).to be_valid
+    end
+  end
+
+  context "when given a string which is too long" do
+    it "validates without errors" do
+      question = described_class.new
+      question.validate
+      question.text = "a" * 5001
```
Comment (aliuk2012) [inline spec/models/question/long_text_spec.rb:37]:
```suggestion
      question = described_class.new(text: "a" * 5001)
```

File: spec/models/question/long_text_spec.rb [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      question.validate
+      expect(question.errors[:text]).to include(I18n.t("activemodel.errors.models.question/long_text.attributes.text.too_long"))
```
Comment (aliuk2012) [inline spec/models/question/long_text_spec.rb:39]:
```suggestion
			expect(question).not_to be_valid
      expect(question.errors[:text]).to include(I18n.t("activemodel.errors.models.question/long_text.attributes.text.too_long"))
```

**Comment by aliuk2012** [comment]
good point @DavidBiddle, might be worth creating the new question type as a component rather than a partial which we could test better in isolation.

File: spec/models/question/long_text_spec.rb (hunk 1 above)
Comment (alice-carr) [reply spec/models/question/long_text_spec.rb:27]: Thanks for your suggestions @aliuk2012 - I'll just commit them now

**Comment by DavidBiddle** [comment]
@aliuk2012 will do a separate PR for turning the partial views into components

**Review by aliuk2012 — APPROVED** [review]

**Pull request merged**